subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CachedTaskExecutionIntegrationTest.groovy

```diff
@@ -435,8 +435,25 @@ class CachedTaskExecutionIntegrationTest extends AbstractIntegrationSpec impleme
         then:
         skippedTasks.empty
         output.contains("Build cache key for task ':compileJava' is ")
-        !output.contains("Appending taskClass to build cache key:")
-        !output.contains("Appending inputPropertyHash for")
+        !output.contains("Appending implementation to build cache key:")
+        !output.contains("Appending input value fingerprint for")
+        !output.contains("Appending input file fingerprints for 'classpath'")
+    }
+
+    def "inputs to the build cache key are only reported when build cache logging is enabled"() {
```

Comment:
How is this new test different to the test just above? https://github.com/gradle/gradle/blob/58293289acd8e6e538b619c5246ab73ff0b1580d/subprojects/core/src/integTest/groovy/org/gradle/api/tasks/CachedTaskExecutionIntegrationTest.groovy#L407-L426

Reply:
I didn't find it because of this weird code structure. Duplication removed.

Reply:
I am pretty sure you reviewed the "weird" code structure back in the day...